pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -1184,9 +1186,9 @@ type BRConfig struct {
 	Cluster string `json:"cluster"`
 	// Namespace of backup/restore cluster
 	ClusterNamespace string `json:"clusterNamespace,omitempty"`
-	// DB is the specific DB which will be backed-up or restored
+	// --- Deprecated --- DB is the specific DB which will be backed-up or restored
```

Comment:
Since BR only supports tableFilter from v4.0.3, we'd better change it to `Deprecated from BR v4.0.3`.